web/src/containers/StateDashboard.js

```diff
@@ -84,75 +124,91 @@ const StateDashboard = (
           <h1 className="ds-h1">
             {t('stateDashboard.title', { state: state ? state.name : '' })}
           </h1>
-          <Instruction source="stateDashboard.instruction" />
+          <Instruction source="stateDashboard.introduction" />
+          {stateStatus === STATE_AFFILIATION_STATUSES.APPROVED && (
+            <Instruction source="stateDashboard.instruction" />
+          )}
           <div className="ds-u-margin-top--5 ds-u-padding-bottom--1 ds-u-border-bottom--2">
             <h2 className="ds-h2 ds-u-display--inline-block">
               {state ? state.name : ''} APDs
             </h2>
-            <Button
-              variation="primary"
-              className="ds-u-float--right"
-              onClick={createNew}
-            >
-              Create new{' '}
-              <span className="ds-u-visibility--screen-reader">APD</span>
-              &nbsp;&nbsp;
-              <Icon icon={faPlusCircle} />
-            </Button>
+            {stateStatus === STATE_AFFILIATION_STATUSES.APPROVED && (
+              <Button
+                variation="primary"
+                className="ds-u-float--right"
+                onClick={createNew}
+              >
+                Create new{' '}
+                <span className="ds-u-visibility--screen-reader">APD</span>
+                &nbsp;&nbsp;
+                <Icon icon={faPlusCircle} />
+              </Button>
+            )}
           </div>
         </div>
       </div>
-      {isPending? <PendingApproval /> : null}
+      {stateStatus === STATE_AFFILIATION_STATUSES.REQUESTED ? (
+        <PendingApproval />
+      ) : null}
+      {stateStatus === STATE_AFFILIATION_STATUSES.DENIED ? (
+        <ApprovalDenied />
+      ) : null}
+      {stateStatus === STATE_AFFILIATION_STATUSES.REVOKED ? (
+        <ApprovalRevoked />
+      ) : null}
       {fetching ? <Loading>Loading APDs</Loading> : null}
-      {!fetching && apds.length === 0 ? (
+      {!fetching &&
+      stateStatus === STATE_AFFILIATION_STATUSES.APPROVED &&
+      apds.length === 0 ? (
         <div className="ds-l-row">
           <div className="ds-l-col--8 ds-u-margin-x--auto ds-u-padding-top--2 ds-u-padding-bottom--5 ds-u-color--muted">
             {t('stateDashboard.none')}
```

Comment:
I think @beparticular can weigh in here but we would show the "Approved" screen here if they have been approved but haven't yet created any APDs